website/content/en/docs/usage/operation-modes.md

```diff
@@ -0,0 +1,308 @@
+user:
+  # Host user information is automatically mirrored
+  name: null  # Will use your host username
+  uid: null   # Will use your host UID
+
+mounts:
+  # Home directory mounted read-only
+  - location: "~"
+    writable: false
+  # Shared writable space
+  - location: "/tmp/lima"
+    writable: true
+
+containerd:
+  user: true  # Rootless containerd enabled by default
+
+# Port forwarding enabled
+# Guest agent enabled
+# Host integration enabled
+```
+
+## Plain Mode
+
+Plain mode disables Lima's host integration features, providing a more traditional VM experience similar to what users might expect from Vagrant or basic virtualization tools.
+
+### Key Features of Plain Mode
+
+#### 1. No Automatic Mounts
+- No home directory mounting
+- No shared directories
+- You must explicitly configure any mounts you need
+
+#### 2. No Port Forwarding
+- Ports are not automatically forwarded from guest to host
+- You must manually configure any port forwarding rules
+- Network access requires explicit configuration
+
+#### 3. No Container Engine Integration
+- containerd is not automatically installed or configured
+- No automatic container runtime setup
+- You must manually install and configure container engines
+
+#### 4. No Guest Agent
+- The Lima guest agent does not run
+- No automatic host-guest coordination
+- Reduced background processes in the guest
+
+#### 5. Minimal Host Integration
+- User information may still be mirrored for SSH access
+- Basic VM lifecycle management still works
+- SSH access is still available
+
+### When to Use Plain Mode
+
+Plain mode is ideal for:
+- **Security-Conscious Environments**: When you want minimal host exposure
+- **Traditional VM Usage**: When you prefer explicit control over all integrations
+- **Learning/Educational**: When you want to understand Linux systems without abstractions
+- **Custom Setups**: When you need full control over the guest environment
+- **Isolation**: When you want the guest to be completely separate from the host
+
+### Enabling Plain Mode
+
+To enable plain mode, set the `plain` field to `true` in your Lima configuration:
+
+```yaml
+# Enable plain mode
+plain: true
+
+# In plain mode, these are automatically disabled:
+# - mounts: []
+# - portForwards: []
+# - containerd.system: false
+# - containerd.user: false
+# - Guest agent services
+```
+
+You can also enable plain mode when creating an instance:
+
+```bash
+limactl create --plain my-plain-vm
+```
+
+## Comparison: Integrated Mode vs Plain Mode
+
+| Feature | Integrated Mode | Plain Mode |
+|---------|--------------|------------|
+| **Home Directory Mount** | ✅ Automatic (`~` read-only) | ❌ None |
+| **Shared Directory** | ✅ `/tmp/lima` writable | ❌ None |
+| **Port Forwarding** | ✅ Automatic | ❌ None |
+| **Container Engine** | ✅ containerd by default | ❌ None |
+| **Guest Agent** | ✅ Running | ❌ Disabled |
+| **User Mirroring** | ✅ Full mirroring | ✅ Basic (for SSH) |
```

Comment:
This does look a bit "odd" in the column, we don't actually need the user account in plain mode do we?

* https://github.com/lima-vm/lima/issues/3844